### DISCUSSION TITLE
Setting up briefkasten ends with Error 500

### DISCUSSION THREAD

**Discussion by protonaut** [ndom91/briefkasten#66] [Q&A]
I just pulled briefkasten with docker and related the database to the deplyed briefkasten-app. 
When I launch the page trough the browser I get 
Error 500: 
"Oops! The page you're looking for isn't here.
You might have the wrong address, or the page may have moved."

Also the following notion appears:
"Application error: a client-side exception has occurred (see the browser console for more information)."

Here ist the link of the page:
https://briefkasten.krealyse.de/

How to solve this to get briefkasten running?

**Comment by protonaut** [comment]
OK, I think the error is caused by the same problem the maintainer already explained:

"I believe this is because you didn't setup any of the NextAuth providers (like google or github login). The latest version of Briefkasten warns about this when launching 👍
Generally all your other changes / steps sound good though."

So my final question changed into: 
How to setup a NextAuth provider for briefkasten?

Is it possible to set it up within the docker-compose.yml as a side-to-side container?

**Reply by n-albrecht** [reply]
I had the same issue setting briefkasten up using portainer. I also got an `Error: Please setup NextAuth.js providers`. Since I'm slefhosting the app in my internal network, I don't need any kind of authorization, I'm not sure if that can be done. Hoping for V2.

**Comment by ndom91** [comment]
The current docs aren't great if yuo're not already a bit familiar with the JS ecosystem, but basically you just have to fill in any one set of the provider environment variables. 

In your environment variables file (when you copied `.env.example` for yourself), you have to setup one of (1) [Github](https://authjs.dev/reference/core/providers/github) (2) [Google](https://authjs.dev/reference/core/providers/google) (3) [AzureAD](https://authjs.dev/reference/core/providers/azure-ad) (4) [Keycloak](https://authjs.dev/reference/core/providers/keycloak) (5) [Authentik](https://authjs.dev/reference/core/providers/authentik) or (6) [SMTP (Email Magic Link)](https://authjs.dev/getting-started/providers/email-tutorial#1-installing-nodemailer), for more details on those, check out the [next-auth (auth.js) docs](https://authjs.dev/getting-started/providers/oauth-tutorial)

- Selfhosting authentication docs: https://v1.docs.briefkastenhq.com/docs/self-hosting.html#authentication
